Cargo.toml

```
@@ -17,6 +17,7 @@ members = [
     "crates/cli/cli/",
     "crates/cli/commands/",
     "crates/cli/runner/",
+    "crates/cli/utils/",
```

should be named util
```suggestion
    "crates/cli/util/",
```